Tests/MessagingPush/HttpTest.swift

```diff
@@ -18,24 +19,23 @@ import XCTest
  */
 open class HttpTest: UnitTest {
     public var runner: HttpRequestRunner?
-    public var userAgentUtil: UserAgentUtil!
+    public var deviceInfo: DeviceInfo!
     public var session: URLSession?
 
     override open func setUp() {
         super.setUp()
 
-        userAgentUtil = diGraph.userAgentUtil
+        deviceInfo = diGraph.deviceInfo
 
         /*
          We don't want to run these tests on a CI server (flaky!) so, only populate the runner if
          we see environment variables set in XCode.
          */
-        if let siteId = getEnvironmentVariable("SITE_ID"), let apiKey = getEnvironmentVariable("API_KEY") {
+        if let writeKey = getEnvironmentVariable("WRITE_KEY") {
             runner = UrlRequestHttpRequestRunner()
-            session = CIOHttpClient.getCIOApiSession(
-                siteId: siteId,
-                apiKey: apiKey,
-                userAgentHeaderValue: userAgentUtil.getUserAgentHeaderValue()
+            session = RichPushHttpClient.getCIOApiSession(
+                key: writeKey,
+                userAgentHeaderValue: deviceInfo.getUserAgentHeaderValue()
```

Comment:
Aren't we using `userAgentUtil` anymore? Because looking at the code, `userAgentHeaderValue: userAgentUtil.getUserAgentHeaderValue()` made more sense to me than `userAgentHeaderValue: deviceInfo.getUserAgentHeaderValue()` 😅

Reply:
We still have user agent class, but what the useragent class was doing was combining device info information with the client information. But since we don't have any client information in the NSE. We just need device info for this purpose